canonical/ubuntu.com
# Implement /credentials/shop/order-thank-you as flask view

## Done

- Removed previous `/credentials/shop/order-thank-you` that was previously a React app
- Implemented in flask view

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Go to `/credentials/shop/order-thank-you`
- The page should function as it did previously with the React implementation

## Issue / Card

Fixes [WD-8190](https://warthogs.atlassian.net/browse/WD-8190)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-8190]: https://warthogs.atlassian.net/browse/WD-8190?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
